src/DotNetDelice/Output.fs

```diff
@@ -40,12 +47,12 @@ let private prettyPrinter printable =
     colorprintfn "│ ├── Is FSF Free/Libre: $green[%b]" printable.IsFsf
     colorprintfn "│ └── Included deprecated IDs: $green[%b]" printable.IsDeprecatedType
     printfn "└─┬ Packages:"
-    printable.Names
+    printable.Packages
     |> Seq.iteri (fun i l ->
            let prefix =
                if i = (printable.Count - 1) then "└"
                else "├"
-           printfn "  %s── %s" prefix l)
+           printfn "  %s── %s (%s)" prefix l.Name l.Url)
```

Comment:
This isn't an ideal output, it results in this:

```
License Expression: MIT
├── There are 3 occurances of MIT
├─┬ Conformance:
│ ├── Is OSI Approved: true
│ ├── Is FSF Free/Libre: true
│ └── Included deprecated IDs: false
└─┬ Packages:
  ├── FSharp.Core (https://licenses.nuget.org/MIT)
  ├── Newtonsoft.Json (https://raw.github.com/JamesNK/Newtonsoft.Json/master/LICENSE.md)
  └── System.ComponentModel.Annotations (https://github.com/dotnet/corefx/blob/master/LICENSE.TXT)
```

I don't want to have the URL output unless it's a package that wasn't able to determine the license. This is the reason the `name` is currently generated inline.

Reply:
Sorry and thank you for the hint. The output should now be fixed.